### DISCUSSION TITLE
Android Auto Wireless blocked

### DISCUSSION THREAD

**Discussion by fabermundi** [celzero/rethink-app#1825] [Q&A] [answered]
Hello,

When using the Rethink App as firewall my AAWireless app can't connect to my wireless Android Auto adapter.
I've searched but did not find a solution for this (except to stop or pause Rethink).
I've tried to activate the bypass option for AAWireless but unfortunately this had no effect.

Is there a way that I can use this app with active firewall of rethink?

Thank you in advance

**Comment by onetimecontributor** [comment]
if that is based on Miracast or similar, check this out https://github.com/celzero/rethink-app/discussions/1810#discussioncomment-11466569

**Comment by fabermundi** [comment, marked as answer]
Thanks for the help.
Found out that not the AAWireless App is the problem but Android Auto. So after excluding Android Auto from Rethink it works without problems.
Could have thought earlier in this...
